app/scripts/components/common/map/style-generators/raster-paint-layer.tsx

```diff
@@ -29,7 +29,7 @@ export function RasterPaintLayer(props: RasterPaintLayerProps) {
 
   const { updateStyle } = useMapStyle();
   const [minZoom] = zoomExtent ?? [0, 20];
-  const generatorId = `zarr-timeseries-${id}`;
+  const generatorId = `raster-timeseries-${id}`;
```

Comment:
a nitpick, but it might make sense to let each parent layer pass the generatorPrefix to keep the original way of signaling which layer is which through generator Id

Reply:
yes that makes sense and I made that change here: https://github.com/NASA-IMPACT/veda-ui/pull/1131/commits/aa6e9b882c66d0544bad9ccbd38a7c95aabf867e. I don't fully understand how the generatorId is used however so I can't really test it (other than it doesn't seem to break anything). Could you advise me on if there is a way to manually test that the generatorId is being set? And how is it being used out of curiosity?

Reply:
As far as I understand, generator Id is used to uniquely identify the layers for mapboxgl. (but the entire generator id is also suffixed by layer-id - which should be unique - so it will be only useful if there are layers with the same ID but with different data types) And it might come handy if I face a messy situation ex. I have to check  layer IDs on the dashboard? It is there for more of a practice rather than actual use at this point.